### DISCUSSION TITLE
Isekai Rakuraku Survival ~Seizon Skill Kyousha no Ore ga Bishoujo Yonin to Class Mujintou Seikatsu~ - SuicaoDex

### DISCUSSION THREAD

**Discussion by giscus** [TNTKien/suicaodex#21] [Manga]
# Isekai Rakuraku Survival ~Seizon Skill Kyousha no Ore ga Bishoujo Yonin to Class Mujintou Seikatsu~ - SuicaoDex

More than ten years have passed since a means of transporting individuals from Earth to another world was developed. High school student Kakeru Satomi was taught survival techniques and wisdom by his mountain-dwelling grandfather. Kakeru was sent to another world together with 100 other adventurers of the same age in order to take part in a training program to become an adventurer, but the portal device that linked Earth and the other world vanished right after their transfer...!

https://suicaodex.com/manga/9b12ac90-9407-4fe8-9de8-c8f4fe65df6f

**Comment by Diablo-Demon** [comment]
Sao truyện drop r admin ơi ra thêm ik
